learning_wavelets/models/learnlet_layers.py

```diff
@@ -135,13 +136,27 @@ def __init__(
         self.kernel_size = kernel_size
         self.skip_connection = skip_connection
         self.wav_analysis = WavAnalysis(coarse=True, n_scales=self.n_scales, **wav_analysis_kwargs)
+        self.n_shearlets = n_shearlets
         constraint = None
         if self.tiling_unit_norm:
             constraint = UnitNorm(axis=[0, 1, 2])
         tiling_prefix = 'details_tiling'
-        self.convs_detail_tiling = [
+        self.convs_detail_tiling_fixed = [
             Conv2D(
-                n_tiling,
+                self.n_shearlets,
+                self.kernel_size,
+                activation='linear',
+                padding='same',
+                kernel_initializer='glorot_uniform',
+                use_bias=tiling_use_bias,
+                kernel_constraint=constraint, 
```

Comment:
This might be a bug: you don't want to use the unit norm constraint on shearlet filters, because they should stay as is.

This constraint might make them evolve during training.

Reply:
```suggestion
                kernel_constraint=None, 
```

Reply:
great!

Reply:
I also removed the constrain for the synthesis shearlet filters